src/res_parsetree_viewer.ml

```diff
@@ -316,7 +341,8 @@ let hasAttributes attrs =
       match attr with
       | ( {
             Location.txt =
-              "bs" | "res.template" | "ns.ternary" | "ns.braces" | "ns.iflet";
+              ( "bs" | "res.async" | "res.await" | "res.template" | "ns.ternary"
```

Comment:
same here

Reply:
Done in ed9b314f529e1d8d6c10a47066baa2489448f334